main.cs

```diff
@@ -1,9 +1,40 @@
 using System;
+
+// Access system.collections.generic to be able to create lists
+using System.Collections.Generic;
 class Program
 {
     public static void Main(string[] args)
     {
-        Console.WriteLine("Hello World");
+
+
+        var Player1 = new Player();
+        Console.WriteLine("Please enter your name");
+        // Create a list to store player inputs in while selecting a name
+        // Create a 'PlayerName' list to store each character a player inputs during         name selection
+        List<char> PlayerName = new List<char>();
+        // While player name is not selected...
+        while (Player1.name != "")
+        {
+            Player1.readInput();
+            Player1.selectName(PlayerName);
+            // // Read the player's input
+            // Player1.readInput();
+            // PlayerName.Add(Player1.keyInput);
+            // Console.WriteLine("\n you pressed " + Player1.keyInput);
+            // // If enter is pressed..
+            // if (Player1.keyInput == (char)(13))
+            // {
+
+            //     //  Convert & Merge each character typed into one string to form a                       name
+            //     // MAYBE MAKE INTO A METHOD TOMORROW
+            //     string playersName = string.Concat(PlayerName);
+            //     Console.WriteLine("You pressed enter! \n" + "Your name is " + playersName);
+            //     //
+            // }
+        }
+      // WEATHER TEST CODE
```

Comment:
Environment test code still exists here, click the dropdown button to see it all